peer.go

```diff
@@ -1562,7 +1561,18 @@ out:
 			// If not already active, we'll add this channel to the
 			// set of active channels, so we can look it up later
 			// easily according to its channel ID.
-			p.activeChannels[chanID] = newChan
+			lnChan, err := lnwallet.NewLightningChannel(
+				p.server.cc.signer, p.server.witnessBeacon,
+				newChan,
+			)
+			if err != nil {
+				p.activeChanMtx.Unlock()
```

Comment:
Missing a call to `lnChan.Stop()` here. Otherwise, we never actually ensure the sigpool goroutines have been stopped.

Reply:
I think if this returns an error, then it is assumed the channel was never started?

Reply:
Correct! (atm)